tensorflow_datasets/object_detection/waymo_open_dataset_test.py

```diff
@@ -25,15 +25,19 @@
 
 
 class WaymoOpenDatasetTest(testing.DatasetBuilderTestCase):
+  """Waymo Open Dataset tests."""
+
   DATASET_CLASS = waymo_open_dataset.WaymoOpenDataset
+  BUILDER_CONFIG_NAMES_TO_TEST = ["v_1_0"]
   SPLITS = {
       "train": 1,  # Number of fake train example
-      "validation": 1,  # Number of fake test example
+      "validation": 1,  # Number of fake validation example
   }
 
   def setUp(self):
+    """Set up Waymo Open Dataset tests."""
     super(WaymoOpenDatasetTest, self).setUp()
-    self.builder._CLOUD_BUCKET = self.example_dir
+    self.builder.builder_config.cloud_bucket = self.example_dir
```

Comment:
Where the tests really passing ? This line is ignored as the dataset is  recreated in `builder = self._make_builder(config=config)`, so running test would generate the full dataset.

I'll fix this internally, as I feel you should move on to new tasks.